keras_cv/layers/preprocessing3d/global_random_dropping_points.py

```diff
@@ -38,14 +38,16 @@ class GlobalRandomDroppingPoints(base_augmentation_layer_3d.BaseAugmentationLaye
       A dictionary of Tensors with the same shape as input Tensors.
 
     Arguments:
-      drop_rate: A float scalar or Tensor sets the probability threshold for dropping the points.
+      drop_rate: A float scalar sets the probability threshold for dropping the points.
     """
 
-    def __init__(self, drop_rate, **kwargs):
+    def __init__(self, drop_rate=None, **kwargs):
         super().__init__(**kwargs)
+        drop_rate = drop_rate if drop_rate else 0.0
+
+        if drop_rate <= 1:
```

Comment:
This should be if drop_rate > 1

(Hopefully we'll have a test failure indicating this?)

Reply:
Sorry for the typo.